### ISSUE TITLE
NTR: C10: decanoylcarnitine high in blood (Lab: chemistry: carnitine compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8559]
New term request C10: decanoylcarnitine high in blood (segal_030514174723): Acylcarnitines

**Comment by pnrobinson** [comment]
@MickeySegal
there is a new term
Elevated circulating decenoylcarnitine concentration
HP:0033439

(note spelling differs in one letter and the above is C10:1  rather than C10).
Would it be correct to label this

Elevated circulating C10 acylcarnitine concentration
as the primary label -- this would match with the other terms

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000273
- label: Elevated circulating decanoylcarnitine concentration
- parents: [Elevated circulating medium-chain acylcarnitine concentration (HP:0035017)](https://hpo.jax.org/app/browse/term/HP:0035017)
- pmid: 
- def: Concentration of decanoylcarnitine (C10:0) in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8559
- synonyms: C10: decanoylcarnitine high in blood(NONE)|Elevated circulating C10:0 acylcarnitine concentration(NONE)
